### ISSUE TITLE
add support for thermostats

### ISSUE THREAD

**Issue by auchter** [auchter/haaska#5]
The new skills API supports commands for thermostats. haaska should support controlling thermostats in Home Assistant via this API.

**Comment by bthome** [comment]
+1  Would be great if it had all the "get_" so you can inquire what the thermostat is set at (e.g. remote homes).

**Comment by auchter** [comment]
> Would be great if it had all the "get_" so you can inquire what the thermostat is set at

I agree, but unfortunately the Smart Home API currently only supports setting, incrementing, and decrementing the temperature: https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/smart-home-skill-api-reference#Temperature%20Control%20Messages

Hopefully Amazon will support this in the future, though.

**Comment by daenny** [comment]
I did a first try for the functionality:
https://github.com/auchter/haaska/pull/28

**Comment by trisk** [comment]
#42 has been merged, this can be closed.
